lib/Models/ArcGisPortalItemReference.ts

```diff
@@ -114,7 +114,10 @@ export class ArcGisPortalItemStratum extends LoadableStratum(
   @computed get rectangle() {
     if (this.arcgisPortalItem === undefined) return undefined;
 
-    if (this.arcgisPortalItem.extent !== null) {
+    if (
+      this.arcgisPortalItem.extent !== null &&
+      this.arcgisPortalItem.extent.length === 2
+    ) {
       return createStratumInstance(RectangleTraits, {
         west: this.arcgisPortalItem.extent[0][0],
         south: this.arcgisPortalItem.extent[0][1],
```

Comment:
Yeah so sometimes a portal item will report an extent like
````
{
   ...
   extent: [],
   ...
}
````
rather than a valid value like
````
{
   ...
   extent: [[0,0], [1,1]],
   ...
}
````

Reply:
Maybe we also need to validate the values, at least to make sure `west` > `east` and `north` > `south`. Otherwise, wrong values might screw up other things.